lib/generators/logidze/install/functions/logidze_capture_exception.sql

```diff
@@ -0,0 +1,26 @@
+CREATE OR REPLACE FUNCTION logidze_capture_exception(error_data jsonb) RETURNS boolean AS $body$
+  -- version: 1
+BEGIN
+  -- Feel free to change this function to change Logidze behavior on exception.
+  --
+  -- Return `false` to raise exception or `true` to commit record changes.
+  --
+  -- `error_data` contains:
+  --   - returned_sqlstate
+  --   - message_text
+  --   - pg_exception_detail
+  --   - pg_exception_hint
+  --   - pg_exception_context
+  --   - schema_name
+  --   - table_name
+  -- Learn more about available keys:
+  -- https://www.postgresql.org/docs/9.6/plpgsql-control-structures.html#PLPGSQL-EXCEPTION-DIAGNOSTICS-VALUES
+  --
+  RAISE USING ERRCODE = error_data->>'returned_sqlstate',
+    MESSAGE = error_data->>'message_text'
```

Comment:
interesting.

Reply:
Oops, I just removed it, `logidze_capture_exception` will be an empty function that returns false by default.